### PR TITLE
Surface view cache tags on subscription tabs.

### LINKED ISSUE #50: Caching issue on the Message Subscribe UI tabs
I'd initially thought this issue was [one with Flag](https://www.drupal.org/node/2638878), but after further testing (and failing to reproduce that issue), I discovered that our page controller is only using cache tags from the flag entity, not the corresponding view.

PR forthcoming.

### PR DESCRIPTION
**Pull request by jhedstrom**
- Fixes #50 
- Includes a test to demonstrate issue/fix.

### CODE REVIEW

File: message_subscribe_email/tests/src/Functional/ViewsTest.php [hunk 1]
```diff
@@ -117,6 +117,15 @@ public function testViews() {
     $this->drupalGet('user/' . $users[2]->id() . '/message-subscribe/subscribe_user');
     $this->assertSession()->pageTextContains($users[1]->label());
     $this->assertSession()->pageTextContains(t("Don't send email"));
+
+    // Login user 3.
+    $this->drupalLogin($users[3]);
+    $this->drupalGet('user/' . $users[3]->id() . '/message-subscribe');
+    $this->assertSession()->pageTextContains(t('You are not subscribed to any items.'));
+    $flag = $this->flagService->getFlagById('subscribe_node');
+    $this->flagService->flag($flag, $node, $users[3]);
```
Comment (amitaibu) [inline message_subscribe_email/tests/src/Functional/ViewsTest.php:127]: maybe worth adding a comment that by seeing the node label, we confirm the Views cache is correct.

Rest looks good.

Comment (jhedstrom) [reply message_subscribe_email/tests/src/Functional/ViewsTest.php:127]: Done!

**Comment by amitaibu** [comment]
(@jhedstrom  I'm not with my computer, so can't push to d.o until the evening, so if you want to push, or wait for me )

**Comment by amitaibu** [comment]
Was pushed to d.o.

**Pull request merged**